src/OpenTelemetry.Api.ProviderBuilderExtensions/Logs/OpenTelemetryDependencyInjectionLoggerProviderBuilderExtensions.cs

```diff
@@ -14,6 +14,9 @@ namespace OpenTelemetry.Logs;
 /// Contains extension methods for the <see cref="LoggerProviderBuilder"/> class.
 /// </summary>
 #if EXPOSE_EXPERIMENTAL_FEATURES
+#if NET8_0_OR_GREATER
```

Comment:
Check both the conditions on the same `#if`?

```suggestion
#if EXPOSE_EXPERIMENTAL_FEATURES && NET8_0_OR_GREATER
```